### PR TITLE
Added an export of orders

### PR DESCRIPTION
**Pull request by enesaktay**

### CODE REVIEW

**Comment by stefandoorn** [comment]
It would be nicer to split the adjustment to the `DateTimeToStringHandler` into a separate PR. Strictly seen it's unrelated to the order exporter and can be handled/tested/merged separately :)

File: src/Resources/config/services_export_csv.yml [hunk 1]
```diff
@@ -17,3 +17,13 @@ services:
...
+
+    sylius.exporter.orders.csv:
+        class: FriendsOfSylius\SyliusImportExportPlugin\Exporter\ResourceExporter
+        arguments:
+            - "@sylius.exporter.csv_writer"
+            - "@sylius.exporter.pluginpool.orders"
+            - ["Id", "Number" ,"Notes", "State", "Checkout_completed_at", "Items_total", "Adjustments_total", "Total", "Created_at", "Updated_at", "Currency_code", "Locale_code", "Checkout_state", "Payment_state", "Shipping_state", "Token_value", "Customer_ip"]
+            - "@sylius.exporters_transformer_pool"
+        tags:
+            - { name: sylius.exporter, type: sylius.order, format: csv }
```
Comment (stefandoorn) [inline src/Resources/config/services_export_csv.yml]: Missing newline here :)

File: src/Resources/config/services_export_excel.yml [hunk 2]
```diff
@@ -17,3 +17,13 @@ services:
...
+
+    sylius.exporter.orders.xlsx:
+        class: FriendsOfSylius\SyliusImportExportPlugin\Exporter\ResourceExporter
+        arguments:
+            - "@sylius.exporter.excel_writer"
+            - "@sylius.exporter.pluginpool.orders"
+            - ["Id", "Number" ,"Notes", "State", "Checkout_completed_at", "Items_total", "Adjustments_total", "Total", "Created_at", "Updated_at", "Currency_code", "Locale_code", "Checkout_state", "Payment_state", "Shipping_state", "Token_value", "Customer_ip"]
+            - "@sylius.exporters_transformer_pool"
+        tags:
+            - { name: sylius.exporter, type: sylius.order, format: xlsx }
```
Comment (stefandoorn) [inline src/Resources/config/services_export_excel.yml]: Same here (missing newline) :)

**Comment by lsmith77** [comment]
one question is how elaborately we want to test each type. i think so far we duplicated the bulk of the tests. not sure if this makes sense in the long run.

File: src/Resources/config/services_export_csv.yml (hunk 1 above)
Comment (enesaktay) [reply src/Resources/config/services_export_csv.yml]: I don't see where where the newline is missing, the first one was already without a newline so i copied the style :)

Comment (enesaktay) [reply src/Resources/config/services_export_csv.yml]: Nvm, @lsmith77 clarified what was said to me, fixed it!

**Comment by enesaktay** [comment]
as the commit message implies I added the userdata and productlist to the order export and removed the WIP in the title as it should now work as wished

**Comment by enesaktay** [comment]
has unwanted commits in it, #107 is the new PR without the unwanted commits

**Pull request closed without merging**